core/drivers/gic.c

```diff
@@ -260,11 +260,11 @@ static void gic_legacy_sync_sgi_config(struct gic_data *gd)
 		/* Ignore matching bits */
 		if (!(BIT32(n) & (grp0 ^ gd->per_cpu_group_status)))
 			continue;
-		need_sync = true;
 		/*
 		 * SGI-n differs from primary CPU configuration,
 		 * let's sync up.
 		 */
+		need_sync = true;
```

Comment:
I forgot to mention my comment also applied to _gicv3_sync_sgi_config_ line 225.

Reply:
I'll fix it.